webflow-hacks/cms-hacks/3/3.ts

```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // get the amount (length) of ".cms-job-counter" classes
+  // and assign that amount to a constant named jobsCounter
+  const jobsCounter = document.querySelectorAll('.cms-jobs-counter').length;
+
+  // select the element with class ".hack3-jobs-amount"
+  const jobsCount: HTMLElement = document.querySelector('.hack3-jobs-amount') as HTMLElement;
```

Comment:
@aexshafii I would add null checks here to make this script safe.
This is the primary reason why we use Typescript 💪🏻 

```typescript
const jobsCount = document.querySelector<HTMLElement>('.hack3-jobs-amount');
if (!jobsCount) return;
```

Reply:
I see. Making the change now. Thanks!